### DISCUSSION TITLE
How do i make it look/work like it used to when it was yellow?

### DISCUSSION THREAD

**Discussion by Nojarosann** [sumatrapdfreader/sumatrapdf#2461] [Q&A]
<img width="960" alt="Screenshot_130" src="https://user-images.githubusercontent.com/100730490/162015804-4634fa9d-e6c6-4508-8a48-70ba78521ac1.png">

It's stuck like this, i don't like this layout of it looking like pages, i really preferred whenit was just a yellow background with words in front and the pages showing on screen just disappeared when i either scrolled once or pressed space.

A few months ago i set it to be gray words with a black background and it was heaven for my eyes, but now i have this, i don't like this how do i change it
I already set the UseFIxedPageUI to true but nothing changes.

**Comment by GitHubRulesOK** [comment]
Don't know what settings you had previous but for yellowish in recent versions of settings you need to decrease the blue value
```
	BackgroundColor = #ffffc0
        GradientColors #ffffc0
```
Note the top area and sidebar colors are controlled by windows (that's a [separate issue ](https://github.com/sumatrapdfreader/sumatrapdf/discussions/2441?notification_referrer_id=NT_kwDOAMKf4bMzMzQ1NzM1NzE5OjEyNzU0OTEz#discussioncomment-2481773)not where SumatraPDF controls viewing content and its background canvas colours)

![image](https://user-images.githubusercontent.com/12754913/162020781-a7251531-64d1-4932-916a-05b1dfa5ebd7.png)

**Reply by Nojarosann** [reply]
Actually my biggest problem is with the appearance. What i don't want is this page-like appearance. What i rather is how the general looks it had when the default color was yellow not the yellow color itself.

I preffered when the pdf showed as a blck of text and when i scrolled down it changed everything onscreen to the next page, I only use Sumatra PDF to read novels offline, the eventual images that pop on screen being page-like wasn't a problem but all of it including the text being like this frustrates me a little

**Reply by GitHubRulesOK** [reply]
That was legacy ebook viewer thus not related to chm or comic nor PDF only eBooks for that you need an older version 3.3 or earlier with `UseFixedPageUI = false`

**Reply by Nojarosann** [reply]
ohh really? thanks. I can download earlier version through the of site? my current version is 3.3.3 so i just need the 3.3 is that right?

**Reply by GitHubRulesOK** [reply]
3.3.3 ebooks should look like this with default setting but PDF will still be default white with zoom and search that's always been the case.

![image](https://user-images.githubusercontent.com/12754913/162031203-cd5811bc-3433-48b5-8ce7-f187efae7d78.png)
![image](https://user-images.githubusercontent.com/12754913/162031709-d7dcb039-f876-47e1-8403-156b7ffec0b7.png)

**Reply by Nojarosann** [reply]
Wait that's strange then. I have the ebook for friday black and it worked fine with that block of text. I also guesses it may have been a problem with the file but when i opened it it had the pages. Why is it then?

**Reply by GitHubRulesOK** [reply]
Without a sample to test  cant say why your experiencing changes however
If I open a PDF that is incorrectly named as an ePub SumatraPDF will detect it and open in the PDF viewer

![image](https://user-images.githubusercontent.com/12754913/162071560-9f464423-99a0-404d-9a44-c1da8db2c0e5.png)
